src/translate/translate.py

```diff
@@ -4,6 +4,9 @@
 import os
 import sys
 import traceback
+from typing import Dict, List, Optional, Tuple, Union
+
+VarValPair = Tuple[int, int]
```

Comment:
These alias declarations are local to the file. If I only include it in sas_task.py, I have to refer to it as sas_task.VarValPair in translate.py or need a `from sas_tasks import VarValPair`. I do not like either because it gives the impression that these var/val pairs always correspond to the ones used in the final sas task. This is not the case in the full mutex groups (using a different encoding in translate.py).

Reply:
Maybe then a solution would be to use different names that mention the difference? Something like `SasVarValPair`, `GroundedVarValPair`, or whatever the difference is.

Reply:
It's really the same thing, just using different encodings.

Reply:
What is different about the encodings? Can this difference be part of the name?

Reply:
This would really only make affairs unnecessarily complicated. Both encodings are computed by the same function (=> implies same return type) but based on different sets of mutex groups.

Reply:
I'm confused: you said that the types are different enough that they should not use a common type but each have their own version of `VarValPair`, but at the same time they are not different enough to give the two types different names?

Reply:
They have the same type and are computed by the same function but with different underlying mutex groups. So var1 of one encoding does not correspond to var1 of the other one. If I refer to this type via sas_tasks.SOMETHING, this gives the impression that they always correspond to the encoding used for the final sas task, which I would find highly confusing.